client/widgets/Images.cpp

```diff
@@ -454,7 +455,15 @@ void CShowableAnim::blitImage(size_t frame, size_t group, SDL_Surface *to)
 	Rect src( xOffset, yOffset, pos.w, pos.h);
 	auto img = anim->getImage(frame, group);
 	if(img)
-		img->draw(to, pos.x, pos.y, &src, alpha);
+	{
+		if(alpha != UINT8_MAX)
+		{
+			const ColorFilter alphaFilter = ColorFilter::genAlphaShifter(vstd::lerp(0.0f, 1.0f, alpha/255.0f));
```

Comment:
Strictly speaking, you don't need lerp here - alpha/255.f already gives you value between 0 and 1
(not an issue, feel free to ignore)